README.md

```diff
@@ -1 +1,282 @@
-# mobile-portfolio-page
\ No newline at end of file
+<a name="readme-top">Mobile Portfolio Project</a>
```

Comment:
- [ ] Please edit the contents of your readme.md file to be personalized to this project. All sections that do not apply to this project and all comments should be removed. This will make your work look more professional.